src/BeneficiariesFile.hs

```diff
@@ -19,13 +19,14 @@ import Text.Read (readMaybe)
 import Prelude hiding (lines, readFile, words)
 
 data Beneficiary = Beneficiary
-  { address :: !PubKeyAddress
+  { srcText :: !Text
```

Comment:
Ideally we'd pretty print the beneficiary using the config, but I'm not sure it's possible to reverse the address stuff, at least without storing the original form. Haven't looked into it too deep. At the moment nothing is modifying Beneficiaries anyway.

It's important to take into account the config because ideally you leave all command line args the same except for which file we point to

Reply:
We shouldn't need to keep the original text? We do have address serialisation functions in this repo. They're even tested to ensure they're reflexive

Reply:
Ah yeah you're right, I was intimidated by the different case for PKH vs address but it seems nothing complicated is going on there